### PR TITLE
Simply formatting of some tagger code.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

File: src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.TagSource_ProduceTags.cs [hunk 1]
```diff
@@ -637,7 +635,10 @@ private void ProduceTagsSynchronously(TaggerContext<TTag> context)
                 {
                     // Otherwise report back on the foreground asap to update the state and let our 
                     // clients know about the change.
-                    RegisterNotification(() => UpdateStateAndReportChanges(newTagTrees, bufferToChanges, newState), 0, cancellationToken);
+                    RegisterNotification(() => UpdateStateAndReportChanges(
```
Comment (sharwell) [inline src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.TagSource_ProduceTags.cs]: :memo: I really don't like this layout. It makes it look like there are 3 arguments to `UpdateStateAndReportChanges`, and one argument to `RegisterNotification`. The following would be a big improvement:

```csharp
RegisterNotification(
    () => UpdateStateAndReportChanges(newTagTrees, bufferToChanges, newState),
    delay: 0,
    cancellationToken: cancellationToken);
```

File: src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.TagSource.cs [hunk 2]
```diff
@@ -80,6 +80,9 @@ private sealed partial class TagSource : ForegroundThreadAffinitizedObject
             public event EventHandler Paused;
             public event EventHandler Resumed;
 
+            public TaggerDelay AddedTagNotificationDelay => _dataSource.AddedTagNotificationDelay;
```
Comment (sharwell) [inline src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.TagSource.cs]: ❔ Why move these? I'm used to seeing properties after constructors.

Comment (CyrusNajmabadi) [inline src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.TagSource.cs]: We generally have fields/simple-props before constructors.  Though we're not hugely consistent.

File: src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.TagSource_ProduceTags.cs (hunk 1 above)
Comment (CyrusNajmabadi) [inline src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.TagSource_ProduceTags.cs]: agreed. that was not formatted as i intended.

**Pull request merged**